source/testcommon/testcommon/etl/read_csv.py

```diff
@@ -0,0 +1,49 @@
+from pyspark.sql import SparkSession, DataFrame
+from pyspark.sql import types as T, functions as F
+
+
+def read_csv(
```

Comment:
I've added a "#comment" column to our test CSV files.
Let's talk about that idea.